experimental/dds/tree2/src/shared-tree-core/branch.ts

```diff
@@ -157,13 +157,13 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 		change: TChange,
 		revision: RevisionTag,
 	): [change: TChange, newCommit: GraphCommit<TChange>] {
-		return this.applyChange(change, revision, LocalCommitSource.Default);
+		return this.applyChange(change, revision, RevertibleKind.Default);
 	}
 
 	private applyChange(
 		change: TChange,
 		revision: RevisionTag,
-		undoRedoType: LocalCommitSource | undefined,
+		revertibleKind: RevertibleKind | undefined,
```

Comment:
It doesn't seem like undefined needs to be allowed. And if it does, maybe it should be treated like Default?